Packs/SplunkPy/ReleaseNotes/3_0_17.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### SplunkPy
+
+- Fixed an issue where proxy parameter wasn't parsed correctly.
```

Comment:
WDYT?
@ShirleyDenkberg 
```suggestion
- Fixed an issue where proxy parameter wasn't parsed correctly in cases which the instance configuration was not checked.
```

Reply:
- Fixed an issue where proxy parameter wasn't parsed correctly in cases it was not checked.